charts/argo/templates/server-deployment.yaml

```diff
@@ -63,6 +63,14 @@ spec:
             value: {{ .Values.server.baseHref | quote }}
           resources:
             {{- toYaml .Values.server.resources | nindent 12 }}
+          volumeMounts:
```

Comment:
I would put the `volumeMounts:` inside the with to so it is not output at all if there are no mounts present, same goes for `volumes` below.

Reply:
done